Cargo.toml

```diff
@@ -340,7 +340,9 @@ once_cell = { version = "1.5.2", default-features = false, features=["std"], opt
 once_cell = { version = "1.5.2", default-features = false, features=["std"] }
 
 [target.'cfg(all(target_arch = "wasm32", target_vendor = "unknown", target_os = "unknown", target_env = ""))'.dependencies]
-web-sys = { version = "0.3.37", default-features = false, features = ["Crypto", "Window"] }
+web-sys = { version = "0.3.51", default-features = false, features = ["Crypto", "Window", "ServiceWorkerGlobalScope"], optional = true }
```

Comment:
Why was this made optional?

Reply:
Looks like in my commit it was optional both before and after 🤔. I guess something changed there and the merge went bad.

Reply:
It's optional on `main`. I think I made a mistake in the merge conflict.

Reply:
Fixed.